ci-cd/build.js

```
@@ -7,43 +7,63 @@ import browserslist from "browserslist";
+}
+fs.cpSync("src", "dist", {
+	recursive: true
+});
+process.chdir("dist");
+
+const baseDir = ".";
 const cssTargets = browserslistToTargets(browserslist(">= 0.1%"));
 
 processDir(baseDir);
 
 let importMapJSON = fs.readFileSync("index.html", "utf-8").match(/<script type="importmap">([^]+?)<\/script>/)[1];
```

Biome (1.9.4)

[error] The regular expression includes this negated empty character class.

Negated empty character classes match anything.
If you want to match against [, escape it \[.
Otherwise, remove the character class or fill it.

(lint/correctness/noEmptyCharacterClassInRegex)